Sources/MockoloFramework/Operations/OutputWriter.swift

```diff
@@ -46,7 +46,7 @@ func write(candidates: [(String, Int64)],
         return ret
     }
 
-    _ = try? ret.write(toFile: outputFilePath, atomically: true, encoding: .utf8)
+    _ = try ret.write(toFile: outputFilePath, atomically: true, encoding: .utf8)
```

Comment:
There are several options to resolve #189, but I thought that not ignoring write error is more versatile than others. Since mockolo command's primary purpose is creating mock file, it seems natural to finish whole process and report this error as the cause when failed to write mock file.